packages/js/src/general/components/sidebar-layout.js

```diff
@@ -15,12 +15,20 @@ export const SidebarLayout = ( { contentClassName, children } ) => {
 	const premiumLinkSidebar = useSelectGeneralPage( "selectLink", [], "https://yoa.st/jj" );
 	const premiumUpsellConfig = useSelectGeneralPage( "selectUpsellSettingsAsProps" );
 	const academyLink = useSelectGeneralPage( "selectLink", [], "https://yoa.st/3t6" );
+	const premiumLinkList = useSelectGeneralPage( "selectLink", [], "https://yoa.st/17h" );
 	const { isPromotionActive } = useSelect( STORE_NAME );
 
 	return (
 		<div className="yst-flex yst-gap-6 xl:yst-flex-row yst-flex-col">
 			<div className={ classNames( "yst-@container yst-flex yst-flex-grow yst-flex-col", contentClassName ) }>
 				{ children }
+				{ isPremium ? null : (
```

Comment:
I tried to prevent this, but I suppose we will cross the "actually, lets create a page with the sidebar but not the upsell list" bridge when we get there 😬